src/Nri/Ui/Button/V5.elm

```diff
@@ -0,0 +1,807 @@
+                    , imageHeight = 15
+                    , shadowHeight = 3
+                    , minWidth = 100
+                    }
+
+                Large ->
+                    { fontSize = 20
+                    , height = 56
+                    , imageHeight = 20
+                    , shadowHeight = 4
+                    , minWidth = 200
+                    }
+
+        sizingAttributes =
+            let
+                verticalPaddingPx =
+                    4
+            in
+            [ Css.minHeight (Css.px config.height)
+            , Css.paddingTop (Css.px verticalPaddingPx)
+            , Css.paddingBottom (Css.px verticalPaddingPx)
+            ]
+
+        widthAttributes =
+            case width of
+                WidthExact pxWidth ->
+                    [ Css.maxWidth (Css.pct 100)
+                    , Css.width (Css.px <| toFloat pxWidth)
+                    , Css.paddingRight (Css.px 4)
+                    , Css.paddingLeft (Css.px 4)
+                    ]
+
+                WidthUnbounded ->
+                    [ Css.paddingLeft (Css.px 16)
+                    , Css.paddingRight (Css.px 16)
+                    , Css.minWidth (Css.px config.minWidth)
+                    ]
+
+        lineHeightPx =
+            case size of
+                Small ->
+                    15
+
+                Medium ->
+                    19
+
+                Large ->
+                    22
+    in
+    [ Css.fontSize (Css.px config.fontSize)
+    , Css.borderRadius (Css.px 8)
+    , Css.lineHeight (Css.px lineHeightPx)
+    , Css.boxSizing Css.borderBox
+    , Css.borderWidth (Css.px 1)
+    , Css.borderBottomWidth (Css.px config.shadowHeight)
+    , Css.batch sizingAttributes
+    , Css.batch widthAttributes
+    , Css.Foreign.descendants
+        [ Css.Foreign.img
+            [ Css.height (Css.px config.imageHeight)
+            , Css.marginRight (Css.px <| config.imageHeight / 6)
+            , Css.position Css.relative
+            , Css.bottom (Css.px 2)
+            , Css.verticalAlign Css.middle
+            ]
+        , Css.Foreign.svg
+            [ Css.height (Css.px config.imageHeight) |> Css.important
+            , Css.width (Css.px config.imageHeight) |> Css.important
+            , Css.marginRight (Css.px <| config.imageHeight / 6)
+            , Css.position Css.relative
+            , Css.bottom (Css.px 2)
+            , Css.verticalAlign Css.middle
+            ]
+        , Css.Foreign.svg
+            [ Css.important <| Css.height (Css.px config.imageHeight)
+            , Css.important <| Css.width Css.auto
+            , Css.maxWidth (Css.px (config.imageHeight * 1.25))
+            , Css.paddingRight (Css.px <| config.imageHeight / 6)
+            , Css.position Css.relative
+            , Css.bottom (Css.px 2)
+            , Css.verticalAlign Css.middle
+            ]
+        ]
+    ]
+
+
+styledName : String -> String
+styledName suffix =
+    "Nri-Ui-Button-V4-" ++ suffix
```

Comment:
This ought to change to `V5`. Ah, maybe I'll do that!

Reply:
Whoopsie, many thanks for fixing!